src/models/userRole.js

```diff
@@ -0,0 +1,21 @@
+export default (sequelize, DataTypes) => {
```

Comment:
Why do you have this here?

Reply:
This was created as a join table for `many to many relationship` between `user` and `role`

Reply:
I am wondering what it is doing in this PR